### ISSUE TITLE
The migrate mappings task should remap mixin strings

### ISSUE THREAD

**Issue by haykam821** [FabricMC/fabric-loom#1035]
If a method named `ExampleClass.doSomething` was renamed to `doNewSomething`, then the mixin would be remapped as such:

```diff
 public class ExampleClassMixin {
 	@Redirect(method = "doFullTask", at = @At(value = "INVOKE", target = "Lcom/developer/ExampleClass;doSomething(Z)V"))
 	public static void modifySomething(boolean value) {
 		if (GlobalCondition.condition()) {
 			return;
 		}
 
- 		ExampleClass.doSomething(!value);
+ 		ExampleClass.doNewSomething(!value);
 	}
 }
```

The method call will be migrated, but not the target of the mixin. Ideally, both would be migrated.

**Comment by SNWCreations** [comment]
I think this issue can be solved by using [CadixDev/MercuryMixin](https://github.com/CadixDev/MercuryMixin).

**Comment by modmuss50** [comment]
Yes, MM likely need updating to our mercury fork as well as looking into what https://github.com/CadixDev/MercuryMixin/issues/18 means. Its been so long I have forgotten what this issue was about.

**Comment by SNWCreations** [comment]
> Yes, MM likely need updating to our mercury fork as well as looking into what [CadixDev/MercuryMixin#18](https://github.com/CadixDev/MercuryMixin/issues/18) means. Its been so long I have forgotten what this issue was about.

I've read that, I think it wanted a method which is similar to Mercury#setGracefulClasspathChecks method,
 it could prevent Mercury from being crashed even if the classpath is incomplete. But enabling this may produce some
 wrong result in code.
